cmd/osbuild-composer/main.go

```diff
@@ -108,7 +108,7 @@ func main() {
 			panic(err)
 		}
 
-		sentryhook := sentrylogrus.NewFromClient([]logrus.Level{logrus.PanicLevel,
+		sentryhook := sentrylogrus.NewEventHookFromClient([]logrus.Level{logrus.PanicLevel,
```

Comment:
This is needed, sentry API is not stable yet and the function was renamed in the newer version.